source/sdk/threading/Thread.ooc

```diff
@@ -165,6 +172,26 @@ Mutex: abstract class {
 
 }
 
+MutexUnsafe: class extends Mutex {
+	init: func
+	lock: override func
+	unlock: override func
+}
+
+MutexGlobal: class extends Mutex {
+	_globalMutex := static Mutex new(MutexType Safe)
+	init: func
+	lock: override func {
+		This _globalMutex lock()
+	}
+	unlock: override func {
+		This _globalMutex unlock()
+	}
+	cleanup: static func {
```

Comment:
It is, unless you count the `dispose()` calls to every `Closure`. 
To be fair, I find only two examples in kean (`ByteBuffer` and `RecycleBin`) of using `clean`. :)
@davidhesselbom ?